site/scripts/prepare-blocks.ts

```diff
@@ -154,37 +148,42 @@ const ensureRepositorySnapshot = async ({
     return repositorySnapshotDirPath;
   }
 
-  const { path: unzipDirPath, cleanup: cleanupUnzipDirPath } = await tmp.dir({
+  const { path: untarDirPath, cleanup: cleanupUntarDirPath } = await tmp.dir({
     unsafeCleanup: true,
   });
 
   try {
-    console.log(chalk.green(`Downloading ${zipUrl}...`));
-    // @todo Consider finding cross-platform NPM packages for curl and unzip commands
+    console.log(chalk.green(`Downloading ${tarballUrl}...`));
+    // @todo Consider finding cross-platform NPM packages for curl and untar commands
     await execa(
       "curl",
-      ["-sL", "-o", path.resolve(unzipDirPath, `repo.zip`), zipUrl],
+      ["-sL", "-o", path.resolve(untarDirPath, `repo.tar.gz`), tarballUrl],
       defaultExecaOptions,
     );
 
+    const untarPath = path.resolve(untarDirPath, "repo");
+
+    console.log(chalk.green(`Creating output directory...`));
+    await execa("mkdir", ["-p", untarPath]);
+
     console.log(chalk.green(`Unpacking archive...`));
-    const unzipPath = path.resolve(unzipDirPath, "repo");
+
     await execa(
-      "unzip",
-      ["-q", path.resolve(unzipDirPath, "repo.zip"), "-d", unzipPath],
+      "tar",
+      ["-xvf", path.resolve(untarDirPath, "repo.tar.gz"), "-C", untarPath],
```

Comment:
Optional: if `tar` has aliases for `-xvf` and `-C`, let’s use them them instead. So we’ll have: `["--option1", "--option2", ....]`.

I usually avoid option shortcuts in all my code and docs, but I have missed this somehow in this file. Shortcuts are more cryptic, so the code is harder to follow and maintain. Shortcuts are still useful in CLI when we type one-off commands from a keyboard.